### PR TITLE
Updated README.md, CONTRIBUTORS.md & CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by antzshrek**
- [x] I have read [Contribution Guidelines](https://github.com/CircleCI-Public/circleci-cli/blob/master/CONTRIBUTING.md).
- [x] I have checked for similar issues and haven't found anything relevant.
- [x] This is not a security issue (which should be reported here: https://circleci.com/security/)

**Here are some helpful tips you can follow when submitting a pull request:**

1. Fork [the repository](https://github.com/CircleCI-Public/circleci-cli) and create your branch from `master`.
2. Run `make build` in the repository root.
3. If you've fixed a bug or added code that should be tested, add tests!
4. Ensure the test suite passes (`make test`).
5. The `--debug` flag is often helpful for debugging HTTP client requests and responses.
6. Format your code with [gofmt](https://golang.org/cmd/gofmt/).
7. Make sure your code lints (`make lint`). Note: This requires Docker to run inside a local job.

**If you have any questions, feel free to ping us at @CircleCI-Public/x-team.**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -2,45 +2,127 @@
...
 ## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
```
Comment (antzshrek) [inline CODE_OF_CONDUCT.md:11]:
```suggestion
We as members, contributors, and leaders pledge to make participation in our
community a harassment-free experience for everyone, regardless of age, body
size, visible or invisible disability, ethnicity, sex characteristics, gender
identity and expression, level of experience, education, socio-economic status,
nationality, personal appearance, race, religion, or sexual identity
and orientation.

```

**Comment by marcomorain** [comment]
Hi @antzshrek can you explain a little bit about the change that you are proposing here please?

Thanks

**Comment by antzshrek** [comment]
Hello @marcomorain the only major update I made was on `CODE_OF_CONDUCT.md`, unfortunately, there were some updates that I made a long time ago but didn't push it, and I don't think those updates are important aside that of  `CODE_OF_CONDUCT.md`.

The update that I made on  `CODE_OF_CONDUCT.md` is updating it from _version 1.4_ to _version 2.0_

**Comment by marcomorain** [comment]
Thanks @antzshrek - sorry for the delay in managing the issues on this repo. Could you rebase/clean up the commit history and commit message please, and I will merge to master.

**Comment by JulesFaucherre** [comment]
Hey, thanks for the PR. We updated the code of conduct to version 2.1 in [this PR](https://github.com/CircleCI-Public/circleci-cli/pull/1011), so closing this one

**Pull request closed without merging**